noxfile.py

```diff
@@ -1,7 +1,10 @@
 import nox
 
 
-@nox.session(python=["3.8"])
+@nox.session(
+    python=["3.8"],
+    reuse_venv=True
```

Comment:
I think it might come in handy.  we've used the flag `-rs` to reuse the session ( ie. `nox -rs test`).  @Edilmo what do you think?